scripts/nginx/qbittorrent.sh

```diff
@@ -18,7 +18,7 @@ location /qbittorrent.downloads {
 DIN
 fi
 
-if [[ ! -f /etc/nginx/apps/qbittorrent.conf ]]; then
+if [[ ! -f /etc/nginx/apps/qbittorrent.conf ]] || ! grep -q "proxy_connect_timeout" /etc/nginx/apps/qbittorrent.conf; then
```

Comment:
I'd just remove the conditionals here. If the scriptis triggered, overwrite the configs. It should apply to the rest of the file too I'd say

Reply:
just removed the conditionals